101-webapp-privateendpoint-vnet-injection/azuredeploy.json

```diff
@@ -0,0 +1,395 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "virtualNetwork_name": {
+      "type": "String",
+      "metadata": {
+        "description": "Name of the VNet"
+      }
+    },
+    "serverFarm_name": {
+      "type": "String",
+      "defaultValue": "ServerFarm1",

```

Comment:
this must be unique within the subscription, so use uniqueString() for a default

Reply:
unique in the RG only